src/input.cpp

```diff
@@ -34,6 +34,7 @@
 #include "color.h"
 #include "point.h"
 #include "sdltiles.h"
+#include <cata_tiles.cpp>
```

Comment:
That's not how include works in c++. You should expose `divide_round_down` by declaring it inside a header, for example `cata_utility.h`, where `divide_round_up` is declared.

Reply:
just made the correction, is it ok now?